apps/sms/views/inbox/test/unit/inbox_test.js

```diff
@@ -667,6 +668,9 @@ suite('thread_list_ui', function() {
 
       assert.isFalse(firstThreadNode.classList.contains('unread'));
       assert.isFalse(secondThreadNode.classList.contains('unread'));
+
+      sinon.assert.neverCalledWith(MessageManager.markThreadRead, 1, true);
```

Comment:
@kumarrishav , I checked that changing this line to `sinon.assert.calledWith(MessageManager.markThreadRead, 1, false);` causes this test to fail.